test_scripts/Defects/5_0/2442_Fix_mem_seg_fault_error_in_jsoncpp.lua

```diff
@@ -0,0 +1,72 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2442
+--
+-- Description:
+-- The root cause of the issue is a stack overflow which is occurring due to recursive call of deeply nested areas 
+-- and objects. We have readValue(..) which calls readObject(..) or readArray(..), which call readValue(...). 
+-- Steps to reproduce:
+-- 1) Generate Json file with more than 50 nestings.
+-- 2) The starting point is a call to parse(..).
+-- Expected:
+-- 1) When recursion threshold is exceeded (50 nestings) SDL should STOP.
```

Comment:
SDL must not stop but `When recursion threshold is
exceeded we return invalid messege`